src/main/java/fi/fmi/avi/converter/tac/geoinfo/GeoUtilsTac.java

```diff
@@ -0,0 +1,551 @@
+                    org.locationtech.jts.geom.Geometry selected = box.intersection(fir);
+                    return jts2PolygonGeometry(selected);
+                } else if (relationOperator2.equals("W")) {
+                    //box with right edge at limit2
+                    coords[2].setX(limit2);
+                    coords[3].setX(limit2);
+                    org.locationtech.jts.geom.Geometry box = fact.createPolygon(coords);
+                    org.locationtech.jts.geom.Geometry selected = box.intersection(fir);
+                    return jts2PolygonGeometry(selected);
+                }
+                break;
+                case "S":
+                // box with top edge at limit
+                coords[1].setY(limit);
+                coords[2].setY(limit);
+                if (relationOperator2.equals("E")){
+                    //box with left edge at limit2
+                    coords[0].setX(limit2);
+                    coords[1].setX(limit2);
+                    org.locationtech.jts.geom.Geometry box = fact.createPolygon(coords);
+                    org.locationtech.jts.geom.Geometry selected = box.intersection(fir);
+                    return jts2PolygonGeometry(selected);
+                } else if (relationOperator2.equals("W")) {
+                    //box with right edge at limit2
+                    coords[2].setX(limit2);
+                    coords[3].setX(limit2);
+                    org.locationtech.jts.geom.Geometry box = fact.createPolygon(coords);
+                    org.locationtech.jts.geom.Geometry selected = box.intersection(fir);
+                    return jts2PolygonGeometry(selected);
+                }
+            }
+        }
+        return null;
+    }
+
+    public static org.locationtech.jts.geom.Geometry PolygonGeometry2jtsGeometry(PolygonGeometry geometry) throws JtsToolsException {
+        String json=toGeoJSON(geometry);
+        Feature feature;
+
+        try {
+            feature = getObjectMapper().readValue(json, Feature.class);
+            return JtsTools.jsonFeature2jtsGeometry(feature);
+        } catch (JsonParseException e) {
+            e.printStackTrace();
+        } catch (JsonMappingException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    public static String toGeoJSON(fi.fmi.avi.model.Geometry g) {
+        PolygonGeometry pg = (PolygonGeometry)g;
+        String polygonAsString="{\"type\": \"Feature\", \"properties\":{},"+
+            "\"geometry\": { \"type\": \"Polygon\", \"coordinates\":[[";
+            List<Double>exteriorRingPositions = pg.getExteriorRingPositions(Winding.CLOCKWISE);
+            int l = exteriorRingPositions.size()/2;
+            for (int i=0; i<l; i++){
+                if (i>0) {
+                    polygonAsString=polygonAsString+",";
+                }
+                polygonAsString=polygonAsString+String.format("[%f, %f]",
```

Comment:
There's already a general comment recommending the use of `Locale` argument for `String.format()`. I'm highlighting this line to prove the point as this line breaks many tests on my system's default `Locale`. 

Using `Locale.ROOT` or `Locale.US` works:
```
String.format(Locale.ROOT, "[%f, %f]",
                        exteriorRingPositions.get(i*2+1),
                        exteriorRingPositions.get(i*2))
```

Reply:
Done